core/templates/services/request-interceptor.service.ts

```diff
@@ -80,6 +80,8 @@ export class RequestInterceptor implements HttpInterceptor {
       }
     }
 
+    RequestInterceptor.checkForNullParams(request.params);
```

Comment:
Just to confirm -- is this only for GET and DELETE requests?

In any case, please add a frontend test to check that null params are OK in POST/PUT requests -- we don't want to bar them from those. Thanks!

Reply:
Done.